cibyl/models/openstack/node.py

```diff
@@ -27,4 +28,45 @@ class Node(Model):
     """
     # To be implemented in future PR
 
-    API = {}
+    API = {
+        'name': {
+            'attr_type': str,
+            'arguments': [Argument(name='--name', arg_type=str,
+                          description="Node name")]
+        },
+        'node_id': {
+            'attr_type': str,
+            'arguments': [Argument(name='--id', arg_type=str,
+                          description="Id provided for the node")]
+        },
+        'status': {
+            'attr_type': str,
+            'arguments': [Argument(name='--status', arg_type=str,
+                          description="Active or inactive status on node")]
+        },
+        'time_created': {
+            'attr_type': str,
+            'arguments': [Argument(name='--time-created', arg_type=str,
+                          description="Time stamp when node was created")]
+        },
+        'role': {
+            'attr_type': str,
+            'arguments': [Argument(name='--role', arg_type=str,
+                          description="Role for the node")]
+        }
+    }
+
+    def __init__(self, name: str, node_id: str,
+                 status: str, time_created: str, role: str):
+        super().__init__({'name': name, 'node_id': node_id, 'role': role,
+                           'status': status, 'time_created': time_created})
+
+    def __str__(self, indent=0):
+        info = f'Node name: {self.name.value}'
+        if self.status:
+            info += f'\n Status: {self.status.value.__str__(indent + 2)}'
+        if self.time_created:
+            info += f'\n Time created: {self.time_created.value.__str__(indent + 2)}'
+        if self.role:
```

Comment:
Use verbosity in this method (like in Job __str__) and print role if verbosity > 0